test/blackbox-tests/test-cases/pkg/lockfile-generation.t/run.t

```diff
@@ -0,0 +1,85 @@
+Simple example of generating a lock file with Dune
+
+Generate a `dune-project` file.
+  $ cat >dune-project <<EOF
+  > (lang dune 3.8)
+  > (package
+  >  (name lockfile_generation_test)
+  >  (depends
+  >    foo
+  >    (bar (>= "0.3"))
+  >   ))
+  > EOF
+
+Run the solver and generate a lock directory.
+  $ dune pkg lock --opam-env=pure --opam-repository=mock-opam-repository
+  Selected the following packages:
+  bar.0.4.0
+  baz.0.1.0
+  foo.0.0.1
+  lockfile_generation_test.LOCAL
```

Comment:
I am a bit confused why the lockfile would contain its own project. Am I missing something?

Reply:
Ah good point there is no reason for the packages defined locally to be present in the lockfile. I'll remove them.